.prettierrc

```
@@ -0,0 +1,4 @@
+{
+  "singleQuote": true,
+  "trailingComma": "none"
```

Personal preference, I guess, but I'd prefer if this is set to `es5`. I find that keeping those trailing commas make diffs easier to parse:

```diff
[
-   'one'
+   'one',
+   'two'
]
```

vs:

```diff
[
  'one',
+ 'two',
]
```

```suggestion
  "trailingComma": "es5"
```